### DISCUSSION TITLE
неработает обход

### DISCUSSION THREAD

**Discussion by Mam1241** [Flowseal/zapret-discord-youtube#1110] [Show and tell]
я устанавил включил от имени администратора и ме выдола такую ошыбку хз что делать все новое подскажыте[](url)
![Screenshot_96](https://github.com/user-attachments/assets/29264666-c0b8-409e-b97e-c5c37bb91390)

**Comment by Vic2077** [comment]
А у меня просто перестал открываться Дискорд вчера с Запретом. Отвлёкся от него минут на 10, обновляю страницу - и всё, больше не грузится. На телефоне с тем же вайфаем с БайДиПиАем работает, на ноуте и телефоне с ВПН работает, а вот с Запретом больше не открывается. Хотя Ютуб при этом с Запретом работает хорошо до сих пор.

**Comment by Flowseal** [comment]
https://github.com/Flowseal/zapret-discord-youtube/discussions/1124
